### ISSUE TITLE
Trouble Implementing UI

### ISSUE THREAD

**Issue by mperejda** [kmullin/resque-sliders#18]
I'm having trouble getting the UI to show up. Can anyone offer a detailed checklist on setting up resque-sliders? Thanks!

**Comment by kolobock** [comment]
did you try https://github.com/kmullin/resque-sliders#resque-web-integration ?

**Comment by mperejda** [comment]
Yes. bundle installed the gem.  Created `resque-web_init.rb` and ran `resque-web resque-web_init.rb` and received `load': cannot load such file -- resque-web_init.rb (LoadError)`.
